src/gtk/animate.cpp

```diff
@@ -11,10 +11,10 @@
 // For compilers that support precompilation, includes "wx.h".
 #include "wx/wxprec.h"
 
-#if wxUSE_ANIMATIONCTRL && !defined(__WXUNIVERSAL__)
-
+#if wxUSE_ANIMATIONCTRL
```

Comment:
Not as if it were really important, but I'm not sure why wxUniv check got removed from here?

Reply:
You had suggested the same change in `animation.h` so I made it here too.